qcodes/tests/mockers/test_simulated_ats_api.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+from qcodes.instrument.mockers.simulated_ats_api import SimulatedAlazarATSAPI
+from qcodes.instrument_drivers.AlazarTech.ATS import AcquisitionInterface
+from qcodes.instrument_drivers.AlazarTech.ATS9360 import (
+    AlazarTech_ATS9360 as ATS9360)
+
+
+
+@pytest.fixture(scope='function')
+def simulated_alazar():
+    driver = ATS9360(
+        'Alazar',
+        api=SimulatedAlazarATSAPI(dll_path='simulated'))
+    with driver.syncing():
+        driver.sample_rate(1_000_000_000)
+        driver.decimation(1)
+        driver.trigger_delay(0)
```

Comment:
is setting decimation and trigger_delay really needed? when i was using a similar simulation class a couple of days ago, only the sample_rate was needed to be set, and that's it.

Reply:
Good point, I will remove it.